src/software/sensor_fusion/sensor_fusion_test.cpp

```diff
@@ -11,7 +11,7 @@ class SensorFusionTest : public ::testing::Test
 {
    public:
     SensorFusionTest()
-        : config(MutableDynamicParameters->getMutableSensorFusionConfig()),
+        : config(std::make_shared<SensorFusionConfig>()),
```

Comment:
We can also delete these lines:

line 749:
```
    config->getMutableOverrideGameControllerFriendlyGoalieId()->setValue(false);
    config->getMutableOverrideGameControllerEnemyGoalieId()->setValue(false);
    config->getMutableFriendlyGoalieId()->setValue(0);
    config->getMutableEnemyGoalieId()->setValue(0);
```

Reply:
removed

Reply:
also lines 689-690, 717-718

Reply:
its probably worth having the values set before the test runs right?